sql/core/src/main/scala/org/apache/spark/sql/execution/HiveResult.scala

```diff
@@ -59,18 +59,6 @@ object HiveResult {
         formatDescribeTableOutput(executedPlan.executeCollectPublic())
       case _: DescribeTableExec =>
```

Comment:
shall we remove the `hiveResultString` completely?

Reply:
The previous purpose of hiveResultString was twofold
1. Keep the schema output of show tables/views consistent with the hive
2. Ensure that the result string is compatible with the hive

This PR intended to modify the first point that the result string sequence is compatible with hive should we keep it?

Reply:
OK so we still need it for `HiveComparisonTest`. Shall we refactor the code so that we only invoke `hiveResultString` in `HiveComparisonTest`?